bindata/scripts/bf2-switch-mode.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
lets add a comment explaining where this script is expected to be used and how.

Reply:
Good point!